elixir/apps/domain/lib/domain/resources/resource/changeset.ex

```diff
@@ -66,26 +66,26 @@ defmodule Domain.Resources.Resource.Changeset do
     changeset
     |> validate_and_normalize_cidr(:address)
     |> validate_not_in_cidr(:address, %Postgrex.INET{address: {0, 0, 0, 0}, netmask: 32},
-      message: "can not contain all IPv4 addresses"
+      message: "cannot contain all IPv4 addresses"
     )
     |> validate_not_in_cidr(:address, %Postgrex.INET{address: {127, 0, 0, 0}, netmask: 8},
-      message: "can not contain loopback addresses"
+      message: "cannot contain loopback addresses"
     )
     |> validate_not_in_cidr(
       :address,
       %Postgrex.INET{
         address: {0, 0, 0, 0, 0, 0, 0, 0},
         netmask: 128
       },
-      message: "can not contain all IPv6 addresses"
+      message: "cannot contain all IPv6 addresses"
```

Comment:
Thanks for fixing all the typos 👍

Reply:
No problem.  I saw one in a file I was already in and fixed it, but then that ended up causing other failures, so I just decided to find all occurrences and fix them.